enclave/server.ts

```diff
@@ -57,13 +57,59 @@ const nStates = new ethers.Contract(
 let b: Board;
 
 /*
- * Adjust internal state based on claimed move & notifies all users to
- * update their local views.
+ * Propose move to enclave. In order for the move to be solidified, the enclave
```

Comment:
great func 

nit. comment's a bit confusing. this is the enclave acknowledging that it knows the shielded information of an intended move correct?

also nit. what do you think about changing the word "propose"? it seems like it's asking for the enclave's approval for whether a move is valid rather than acknowledging an intended move

Reply:
hmm what about "claimMove," or just "getSignature" as alternative names?

Reply:
that function really just gets the enclave's signature. in the future the move will be added to a queue so that the client never has to ask the enclave for an update later. but this can't happen until the alchemy node

Reply:
both suggestions make sense to me, ty ty